{{ .ProjectSnake }}/README.bazel.md

```diff
@@ -44,12 +44,8 @@ If the package is not already a dependency of the project, you'll have to do som
 ```shell
 # Update dependencies table to include your new dependency
 % vim pypyroject.toml
-# Update requirements_lock.txt to pin this dependency
-% aspect run requirements.update
-# Update gazelle_python.yaml to list the imports provided by this package
-% aspect run gazelle_python_manifest.update
-# Repeat
-% aspect configure
+# Update lock files to pin this dependency
+% ./repin.sh
```

Comment:
I don't want users to get in the habit of running this in place of `aspect configure` "any time you change deps" because it's so much slower.

> name like "sync_deps" be a little clearer than "repin"

There is no good name. We want to follow the principle of least surprise, but are combining a concept across all languages, and they all have a different name. Python calls it "compile", Nodejs calls it "install", rules_rust calls it CARGO_BAZEL_REPIN, rules_jvm_external calls it `REPIN=1 bazel run @maven//:pin`, Go calls it "tidy", etc.